ProcessLib/LIE/Common/PostUtils.cpp

```diff
@@ -16,11 +16,67 @@ namespace ProcessLib
 {
 namespace LIE
 {
+
+namespace
+{
+bool includesNodeID(std::vector<MeshLib::Node*> const& vec_nodes,
+                       std::size_t node_id)
+{
+    auto itr2 = std::find_if(
+        vec_nodes.begin(), vec_nodes.end(),
+        [&](MeshLib::Node const* node) { return node->getID() == node_id; });
+    return (itr2 != vec_nodes.end());
+}
+
+std::vector<int> const& getmatids(
```

Comment:
longer names are preferred and use camelCase for function names, *e.g.* `getMaterialIdsForNode(...)`

Reply:
done